app/assets/stylesheets/layout.scss

```diff
@@ -302,6 +302,7 @@ td, .table-list li { font-size: 14px; }
 .user-right-content-box { width: calc(100% - 200px); }
 .icon-right-content-box { width: calc(100% - 150px); }
 .character-right-content-box { width: calc(100% - 175px); }
+#user-avatar { text-align: center !important; }
```

Comment:
Is the `!important` necessary? I try to avoid it wherever possible. (I assume this is taken from, like, `.centered` or something)

Reply:
...huh i thought #user-avatar was there to start with. I might just flip that back to using .centered. (and yeah centered does that)

Reply:
It seems sensible to start extracting `.centered` into the relevant locations, especially if we can avoid `!important` in those areas, because I really really dislike `!important`. xD Is `#user-avatar` in another file, maybe?

Reply:
it's in users #show and did apparently pre-exist, just as a class that doesn't seem to have had anything attached? ...uh, unless it had javascript attached

Reply:
After a quick look, it seems like it's `.user-avatar`, and it's only used on `users#show` – there doesn't seem to be any JavaScript interacting with it, so this change should be good without the `!important`,